Editor/SerializableInterfacePropertyDrawer.cs

```diff
@@ -29,6 +26,7 @@ private void Initialize(SerializedProperty property)
             activeDrawer = null;
             serializedProperty = property;
             genericType = GetGenericArgument();
+
```

Comment:
Could you explain the reason for this added blank line?

Reply:
It seems my Rider thought it looked better :) I'll remove it!